pkg/daemon/daemon.go

```diff
@@ -559,6 +562,38 @@ func (d *Daemon) ClearVestige() {
 	d.ResetClient()
 }
 
+func (d *Daemon) AddCgroup(pid int) error {
```

Comment:
I think we should implement a cgroup manager and add the daemon to it, instead add the method `AddCgroup`, `DeleteCgroup` on `*Daemon`. One reason is that maybe we need to support containerized snapshotter/nydusd in the future.

Reply:
Got it. I will refactor this later.

Reply:
Agree. We should support resources limitation based on both raw cgroup controller and containers